go/vt/sqlparser/ast.go

```diff
@@ -226,6 +226,11 @@ type Select struct {
 	Lock        string
 }
 
+// Stream represents a SELECT statement.
+type Stream struct {
+	Select
```

Comment:
It's better to list the members here. That way, we can be more explicit about the constructs we don't support (like where clauses, etc).

In the future, if this becomes as complete as `Select`, we may be better off dropping this type and just adding a flag to `Select` indicating it's a stream.

Reply:
I started with adding an `action` field to `Select` like `Insert/Replace`, but didn't want to cause any performance regressions. I was more interested in inheriting the `Select` methods, but I can go ahead and copy them and reduce the functionality.